include/solana/base64.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include <string>
+
+///
+/// Adapted from https://stackoverflow.com/a/37109258/3626093
+
```

Comment:
kinda stupid we use our own base64, i noticed that websocketpp has it's own implementation. might be worth a quick benchmark to see if theirs is slower. I just briefly glanced at the code and suspect it does have quadratic rather then linear decoding complexity which would be absolutely dumb and something we should probably fix upstream.